src/highlight.js

```diff
@@ -43,11 +43,11 @@ function createElement({ node, style, useInlineStyles, key }) {
 }
 
 function getLineNumberString(lines, startingLineNumber) {
-  return lines.reduce((lineCountString, _, i) => lineCountString + `${i + startingLineNumber}\n`, '');
+  return lines.map((_line, i) => { return <span className="line" key={"line-" + i}>{`${i + startingLineNumber}\n`}</span> })
```

Comment:
And for the className can we use something more unique, maybe scope it to the library? 
`className='react-syntax-highlighter-line-number`